### DISCUSSION TITLE
无法修改反向代理网站配置

### DISCUSSION THREAD

**Discussion by romanticgrass** [1Panel-dev/1Panel#5023] [Q&A]
因为重装系统，我之前把opt/1panel整个目录备份了，重装系统之后，装了1panel之后，我先关闭1panel，然后通过ftp将新装系统的1panel目录删除之后再把原来备份的数据全部还原到opt/1panel，再次开始1panel，配置和以前一样，以前配置的反向代理网站也可以正常反正，但是遇到一个问题，无法修改反向代理配置了，提示访问超时，日志里显示命令执行超时！请问该如何解决？

**Comment by zhengkunwang223** [comment]
你当前的 1panel 版本 和 openresty 版本分别是多少

**Reply by romanticgrass** [reply]
都是最新版本

**Comment by romanticgrass** [comment]
> 你当前的 1panel 版本 和 openresty 版本分别是多少

都是最新版本，请问有解决办法吗，谢谢。

**Reply by zhengkunwang223** [reply]
所有操作都超时，还是只有反代超时？有日志截图吗
